### PR TITLE
Logic

### PR DESCRIPTION
**Pull request by santiago-rodrig**
# Changes

- **created** logic folder under src
- **created** files operate.js and calculate.js under logic folder
- **created** test folder at the root level of the project
- **created** tests calculate.test.js and operate.test.js
- **wrote** content for operate.test.js
- **wrote** content for calculate.test.js
- **wrote** content for operate.js
- **wrote** content for calculate.js
- **modified** App component to import calculate module, note that instead of doing that specifically I just left a comment with the code, this is because **eslint will complain** If not

If there is something missing or wrong please let me know and I'll fix it.

Thanks for your time.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    case 'X':
+      result = bigOne.times(bigTwo);
+      break;
+    case '÷':
+      result = bigOne.div(bigTwo);
+      break;
+    default:
+      result = 'NaN';
+      break;
+  }
```
Comment (bellom) [inline src/logic/operate.js:25]: I think we are still missing some switch cases like;
1. `0-9` and `.` (Note: if you click `.` we should get 0. on the display area)
2. `A/C`

Comment (santiago-rodrig) [reply src/logic/operate.js:25]: Not really, operate should only work for operations, while calculate should only mutate the calculator object with the rest of the buttons.

**Comment by santiago-rodrig** [comment]
# Changes

- **modified** calculate.test.js to include tests for digits, decimal point, and AC
- **modified** calculate.js to respond to digits, decimal point, and AC

If there are still things to solve, please let me know.

Thanks for your time.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      result = bigOne.plus(bigTwo);
+      break;
+    case '-':
+      result = bigOne.minus(bigTwo);
+      break;
+    case 'X':
+      result = bigOne.times(bigTwo);
+      break;
+    case '÷':
+      result = bigOne.div(bigTwo);
```
[the remark this answers is not in the document]
Comment (santiago-rodrig) [reply src/logic/operate.js:20]: You are totally right, what a mistake from my part!

File: src/logic/operate.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      break;
+    default:
+      result = 'NaN';
```
[the remark this answers is not in the document]
Comment (santiago-rodrig) [reply src/logic/operate.js:23]: What you're saying makes sense, I'll implement the changes.

**Pull request merged**